src/MobileBookingHeader/MobileUserDetail.js

```diff
@@ -4,8 +4,9 @@ import * as React from 'react';
 import css from 'styled-jsx/css';
 import { Text } from '@kiwicom/orbit-components';
 
-import { UserContext, type UserContextType } from '../context/User';
+import { UserContext } from '../context/User';
 import SignOutButton from '../common/layout/SignOutButton';
+import type { UserContextType } from '../context/User';
 
 const MobileUserDetailStyle = css`
```

Comment:
For the user detail section of the mobile header when the user uses simple token we don't have the name of the user account, we only have the names of the passengers in the booking.

What should we show instead? 

full login:
![screenshot](https://i.gyazo.com/a14b069abeba606dbca13372a9032935.png)

simple token login: 
![screenshot](https://i.gyazo.com/cc7097ba7c80615753dd8b9efdcd7aa7.png)

@jaroslav-kubicek

Reply:
How about hiding the entire tab altogether? Does the logout button have a purpose?

Reply:
From the two options I prefer the second one.

Reply:
good point! you shouldn't be available to sign out, because you're not fully logged in

Reply:
Well, just make sure also that it's not possible to compromise user data. What if I log in on a publicly used device? Will an unauthorized person be able to download my boarding passes/tickets and see my personal data on them?

Reply:
Maybe it's FE's responsibility though